flake.nix

```diff
@@ -0,0 +1,27 @@
+{
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+  };
+
+  outputs = {nixpkgs, ...}: let
+    system = "x86_64-linux";
+    pkgs = nixpkgs.legacyPackages.${system};
+  in {
+    devShells.${system}.default = pkgs.mkShell {
+      nativeBuildInputs = with pkgs; [
```

Comment:
question (non-blocking): currently dioxus-cli needs to be installed globally with cargo **and** match the dioxus version in Cargo.toml. should we include it in the list of dependencies here?